res/sample/configuration.toml

```diff
@@ -14,39 +14,55 @@ LogLevel = "INFO"
 
   # ExecutionOrder specifies which functions to run and the order to run them.
   # All functions listed must have an entry below in the Pipeline.Functions section
-  ExecutionOrder = "FilterByProfileName, FilterByDeviceName, FilterByResourceName, Transform, SetResponseData"
+  ExecutionOrder = "FilterByProfileName, FilterByDeviceName, FilterByResourceName, TransformXml, SetResponseData"
+
+  # Pipeline.PerTopicPipelines sections allow for multiple pipleline to be defined to get executed when the specified
+  # Topic match the incoming topic. The Id must be unique, but the Topic does not.
+  [Writable.Pipeline.PerTopicPipelines]
+    [Writable.Pipeline.PerTopicPipelines.float]
+      Id = "float-pipeline"
+      Topic = "edgex/events/device/#/Random-Float-Device/#"
```

Comment:
should we add some comments on how this is specified like `edgex/events/{deviceName}/?/{ServiceKey}/?`

Reply:
I'll add a link to the documentation which describes that.

Reply:
Added.